reana-ui/src/pages/launchOnReana/LaunchOnReana.js

```diff
@@ -172,67 +165,73 @@ export default function LaunchOnReana() {
+                  <div>
+                    <div className={styles.name}>
+                      {query.get("name") ?? DEFAULT_WORKFLOW_NAME}
+                    </div>
+                    <a
+                      className={styles.url}
+                      href={query.get("url")}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      {query.get("url")}
+                    </a>
+                  </div>
+                  <section className={styles.actions}>
+                    <Button
+                      primary
+                      icon="rocket"
+                      content="Launch"
+                      onClick={handleLaunch}
+                      disabled={loading || isMissingRequiredParams(query)}
+                    />
+                  </section>
                 </div>
-                <a
-                  className={styles.url}
-                  href={query.get("url")}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {query.get("url")}
-                </a>
                 <div>
-                  <Icon name="file code outline" />
+                  <div className={styles["launch-section-title"]}>
+                    <Icon name="file code outline" />
+                    Specification
+                  </div>
                   {query.get("specification") ?? DEFAULT_SPEC_FILENAME}
                 </div>
                 {query.get("parameters") && workflowParameters && (
                   <div className={styles.parameters}>
-                    <Icon name="sliders horizontal" />
-                    Parameters:
-                    <ul>
-                      {Object.entries(workflowParameters).map(([k, v]) => (
-                        <li key={k}>
-                          {k}: {v}
-                        </li>
-                      ))}
-                    </ul>
+                    <div className={styles["launch-section-title"]}>
+                      <Icon name="sliders horizontal" />
+                      Parameters
+                    </div>
+                    <Table celled>
+                      <Table.Body>
+                        {Object.entries(workflowParameters).map(
+                          ([key, value]) => (
+                            <Table.Row key={key}>
+                              <Table.Cell>{key}</Table.Cell>
+                              <Table.Cell
+                                className={styles["code-snippet-table-cell"]}
+                              >
+                                <CodeSnippet>
+                                  {JSON.stringify(value, null, 2)}
+                                </CodeSnippet>
+                              </Table.Cell>
+                            </Table.Row>
+                          ),
+                        )}
+                      </Table.Body>
+                    </Table>
```

Comment:
This table feels a bit too big, what do you think? Should we remove some of the padding to the cells, and maybe even reduce the font size?